config/default.yml

```diff
@@ -3082,6 +3082,9 @@ Naming/PredicateMethod:
     - call
   AllowedPatterns: []
   AllowBangMethods: false
+  # Methods that are known to not return a boolean value, despite ending in `?`.
+  NonBooleanPredicates:
```

Comment:
I'm fine with the PR in general, I'm only wondering if we can come up with a better name for this, as predicates are supposed to return booleans, so it sounds odd when I read it. I'm thinking of something more fun like:

- `OutlawPredicates`
- `DeviantPredicates`
- `WaywardPredicates`

:D

Reply:
Haha alright, I've changed it to `WaywardPredicates` now.